sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -703,13 +720,13 @@ public WritableColumnVector arrayData() {
 
   public abstract int getArrayOffset(int rowId);
 
-  @Override
-  public WritableColumnVector getChild(int ordinal) { return childColumns[ordinal]; }
-
   /**
-   * Returns the elements appended.
+   * Returns the offset of a struct element at 'rowId' in the child vectors of this.
    */
-  public final int getElementsAppended() { return elementsAppended; }
```

Comment:
Do we need to remove this API inevitably? To be safe, I hope we undo this removal from this `Improve` PR (if possible).
When we start to remove somethings, the review becomes tricky.

Reply:
Oops I removed this by accident. Seems it isn't used anywhere though.